src/css-image-value.js

```diff
@@ -0,0 +1,39 @@
+(function(internal, scope) {
+
+  function CSSImageValue(image) {
+    if (!(image instanceof Image)) {
+      throw new TypeError("image must be an Image object");
+    }
+    this._image = image;
+
+    var imagesEventsHandler = {
+      onload: function() {
```

Comment:
There's no need to wrap these functions inside another object. You can just write

```javascript
function CSSImageValue(image) {
  ...
  function onLoad() {
  ...
  }
  ...
  this._image.onload = onLoad.bind(this);
```